lib/foreman_theme_satellite/documentation.rb

```diff
@@ -82,8 +82,13 @@ module Documentation
         'Products_and_Repositories_content-management' => "#{ForemanThemeSatellite.documentation_root}/managing_content/importing_content_content-management#Products_and_Repositories_content-management",
       },
       'Managing_Hosts' => {
-        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts",
-      }
+        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_managing-hosts",
```

Comment:
Is this change necessary? If it's help for global registration, we should probably prefer the link to global registration directly.

Reply:
My bad, that happened accidentally